wd_core.au3

```diff
@@ -1354,8 +1354,8 @@ EndFunc   ;==>__WD_Get
 ; Syntax ........: __WD_Post($sURL, $sData)
 ; Parameters ....: $sURL  - Location to access via WinHTTP
 ;                  $sData - String representing data to be sent
-; Return values..: Success - Response from web driver.
-;                  Failure - Response from web driver, sets @error to one of the following values:
+; Return values..: Success - Raw response from web driver.

```

Comment:
I don't think adding the word `raw` adds any value here. Maybe it would be better to say "Response from web driver in JSON format" or something similar.

Reply:
This is not always JSON format, for example take a look:

```
			If $iErr Then
				$iResult = $_WD_ERROR_SendRecv
				$sResponseText = $WD_WinHTTPTimeoutMsg
			Else

```

Reply:
But I get your point. Maybe you some further proposals ?

Reply:
`"Response from web driver in JSON format"`
Do I should use this `template` in entire UDF ?

Reply:
I think that should be fine.

Reply:
Sorry for the gross lack of my language skills:

Fine is current version or whether we should start using consistent nomenclature

Because we currently have 3 different forms

`Raw response from web driver`
`Response from web driver`
`Response from web driver in JSON format`

Reply:
I think ` Response from web driver in JSON format` is the best of that group

Reply:
ok